### DISCUSSION TITLE
Downloading YT videos reserved to subscribers

### DISCUSSION THREAD

**Discussion by mario86135** [aclap-dev/video-downloadhelper#938] [Help and Q&A]
### Your question

Hi, I would like to know if, by buying the lifetime license of Video DownloadHelper, I will be able to download YT videos reserved to subscribers (I mean a paid subscription). Maybe, firstly I have to subscribe to the YT channel in question, in order to let Video DownloadHelper detect the video I would like to download?

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: it
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "download_directory": "C:\\Users\\Dell\\dwhelper",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive"
        }
    },
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 2,
    "successfull_dl": 6
}

**Comment by jcfrog** [comment]
hi,
there is no difference in the engine between free and premium. The best thing to do is to test before purchase.

**Comment by Mattmjs** [comment]
Just in case you need to know, youtube downloading is disabled on chrome. Use Firefox or Edge instead.

**Comment by GrampaWildWilly** [comment]
There have been reports on here (you can find them if you search for them) of people with YouTube accounts being unable to download from YouTube while they are logged in, but having no problem if they log out of Google, which means logging out of all Google properties, YouTube being one such property.  There have also been reports of problems downloading age-restricted YouTube content.  This is, of course, only on Firefox because, as Matt pointed out, Chrome is not permitted to download from YouTube.

But as @jcfrog Jérôme suggested, you should experiment.  Be adventurous.  Instead of being the one asking questions, you should be the one reporting the results of your experiments.  As he also said, getting a VDH license will not make a difference to how VDH works on any given site.  If VDH works on a site when you don't have a license, it will still work on the site after you get a license.  If VDH fails on a site when you don't have a license, it will still fail after you get a license.  So don't waste your money.  Test things first.  Only after you have tested things, then decide based on your results whether to buy a license.

**Comment by Albatros78** [comment]
You can use directly some online downloaders for YT videos

https://www.freemake.com/fr/free_video_downloader_exemplary/
https://fr.savefrom.net/9-how-to-download-facebook-video-139.html
https://www.y2mate.com/fr881
